crates/ruff_diagnostics/src/fix.rs

```diff
@@ -10,21 +10,21 @@ use crate::edit::Edit;
 pub enum Applicability {
     /// The fix is definitely what the user intended, or maintains the exact meaning of the code.
     /// This fix should be automatically applied.
-    Automatic,
+    Automatic = 2,
 
     /// The fix may be what the user intended, but it is uncertain.
     /// The fix should result in valid code if it is applied.
     /// The fix can be applied with user opt-in.
-    Suggested,
+    Suggested = 1,
 
     /// The fix has a good chance of being incorrect or the code be incomplete.
     /// The fix may result in invalid code if it is applied.
     /// The fix should only be manually applied by the user.
-    Manual,
+    Manual = -1,
```

Comment:
Nit: `-1` feels weird and I wonder if Rust change's the underlying type because of it. You can change your tests to use less than instead of greater than instead of assigning explicit values. 

Would you, in turn, document the ordering of the values